src/main/java/org/folio/service/holdings/AbstractLoadServiceFacade.java

```diff
@@ -98,14 +99,14 @@ public void loadHoldings(LoadHoldingsMessage message) {
   protected abstract CompletableFuture<Void> loadHoldings(LoadHoldingsMessage message, LoadService loadingService);
 
   private CompletableFuture<HoldingsStatus> populateHoldingsIfNecessary(LoadService loadingService) {
-    return waitForStatus(statusRetryCount, loadingService).thenCompose(loadStatus -> {
+    return getLastLoadingStatus(loadingService).thenCompose(loadStatus -> {
```

Comment:
So, module does not wait for completed status now? How then it work? When the loading is starting?

Reply:
No, It's not waiting for status before starting the job.